pkg/kfapp/kustomize/kustomize.go

```diff
@@ -1462,3 +1463,25 @@ func GenerateYamlWithOperatorAnnotation(resMap resmap.ResMap, instance *unstruct
 	}
 	return buf.Bytes(), nil
 }
+
+func getParameterValue(value string) string {
+	// Check if there is a environment variable
+	// Supported format: $VAR and ${VAR}
+	var re = regexp.MustCompile(`(?m)^\$({(.*)}|\w)`)
+	envVariable := ""
+	if re.Match([]byte(value)){
+		if strings.HasPrefix(value, "${"){
+			envVariable = value[2:len(value)-1]
+		}else if  strings.HasPrefix(value, "$") {
+			envVariable = value[1:]
+		}else {
+			return value
+		}
```

Comment:
It works great now! Only one question, is this `return value` really needed?

```suggestion
		}
``` 

Running the coverage of this function I found that if I remove it we can achieve 100%

```shell
# With return value
$ go test -run TestGetParameterValue -v -covermode=count -coverprofile coverage ./pkg/kfapp/kustomize/...
$ go tool cover -func=coverage
github.com/kubeflow/kfctl/v3/pkg/kfapp/kustomize/kustomize.go:1467:	getParameterValue			90.9%

# After applying the suggestion
$ go test -run TestGetParameterValue -v -covermode=count -coverprofile coverage ./pkg/kfapp/kustomize/...
$ go tool cover -func=coverage
github.com/kubeflow/kfctl/v3/pkg/kfapp/kustomize/kustomize.go:1467:	getParameterValue			100.0%
```